packages/appstore/src/components/cfds-listing/cfds-listing.scss

```diff
@@ -1395,11 +1395,25 @@
         &__pre-appstore {
             grid-template-columns: 11rem 13.4rem 12.6rem 10rem 13.4rem 12.6rem 10rem 10rem 13.5rem;
         }
+        // af, mg, pk
+        &--other-country {
+            grid-template-columns: 11rem 23.4rem 12.6rem 20rem 13.4rem 12.6rem 13.4rem 20rem 17.5rem;
+        }
+
         &--leverage {
             grid-template-columns: 11rem 38rem 36rem 10rem;
             &__pre-appstore {
                 grid-template-columns: 11rem 36rem 36rem 10rem 13.5rem;
             }
+            &__af {
+                grid-template-columns: 11rem 36rem 33.4rem 12.6rem 13.5rem;
+            }
+            &__pk {
+                grid-template-columns: 11rem 36rem 46rem 13.4rem 17.5rem;
+            }
+            &__mg {
+                grid-template-columns: 11rem 36rem 33.4rem 12.6rem 13.4rem;
```

Comment:
We should not add css code based on a country code!

Reply:
Yes it is not best practice but for now this is the only way to patch, instead of revamping the whole css